src/wrapper.js

```diff
@@ -0,0 +1,66 @@
+import { getCurrentWeather } from './api';
+
+async function getWeatherData(place) {
+  const currentWeatherData = await getCurrentWeather(place);
+  return currentWeatherData;
+}
+
+function roundNumbers(data) {
+  for (const [key, value] of Object.entries(data)) {
+    if (typeof value === 'number') {
+      data[key] = Math.round(value);
+    }
+  }
+  return data;
+}
+
+function weatherDataWrapper(current) {
+  const place = current.name;
```

Comment:
Instead of creating many consts, just return `return { place: current.name, country: current.sys.country....}`. Also, destructure current before that